safeeyes/safeeyes/config/lang/es.json

```diff
@@ -0,0 +1,42 @@
+{
+    "meta_info": {
+        "language_name": "Español",
+        "language_name_en": "Spanish"
+    },
+    "exercises": {
+        "long_break_exercises": [
+            "Vete a andar un poco",
+            "Reclínate sobre tu silla y relájate"
+        ],
+        "short_break_exercises": [
+            "Cierra bien tus ojos",
+            "Mueve tus ojos en círculos",
+            "Mueve tus ojos en círculos en el sentido horario",
+            "Mueve tus ojos en círculos en el sentido antihorario",
+            "Parpadea tus ojos",
+            "Bebe un poco de agua"
+        ]
+    },
+    "messages": {
+        "ready_for_a_break": "Listo para una pausa en {} segundos",
+        "safe_eyes_is_disabled": "Safe Eyes está desactivado",
+        "next_break_at_noon": "Próxima pausa a {}",    
```

Comment:
Google suggests `"next_break_at_noon": "Próxima pausa a las {}"` instead of `"next_break_at_noon": "Próxima pausa a {}"`. Could you please verify this?